.github/workflows/push.yml

```diff
@@ -726,6 +726,16 @@ jobs:
             }
           }
           BODY
+      - name: Generate islandis feature-deployment charts for ArgoCD
+        env:
+          FEATURE_NAME: ${{ needs.pre-checks.outputs.FEATURE_NAME }}
+          DOCKER_TAG: ${{ needs.prepare.outputs.DOCKER_TAG }}
+          IMAGES: ${{ needs.prepare.outputs.IMAGES }}
+          GIT_BRANCH_DEPLOY: ${{ needs.pre-checks.outputs.GIT_BRANCH_DEPLOY }}
+        run: |
+          ./infra/src/feature-env.ts values --chart islandis --feature "$FEATURE_NAME" --images "$IMAGES" > ./charts/islandis/$FEATURE_NAME.yaml
+          cat charts/islandis/$FEATURE_NAME.yaml
```

Comment:
Why `cat`? If you need debugging, why not move this to a script with proper logging?